src/main/java/hospital/userservice/dto/response/UserResponse.java

```diff
@@ -0,0 +1,37 @@
+package hospital.userservice.dto.response;
+
+import hospital.userservice.model.Department;
+import hospital.userservice.model.Permission;
+import hospital.userservice.model.enums.Profession;
+import hospital.userservice.model.enums.Title;
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+
+@NoArgsConstructor
+@Getter
+@Setter
+public class UserResponse {
+
+    private Long id;
+    private UUID lbz = UUID.randomUUID();
```

Comment:
I would change "lbz" field to something in english

Reply:
I changed it to employeId.